pkg/docker/config/config.go

```diff
@@ -76,12 +77,15 @@ func GetAuthentication(sys *types.SystemContext, registry string) (string, strin
 		legacyFormat := path == dockerLegacyPath
 		username, password, err := findAuthentication(registry, path, legacyFormat)
 		if err != nil {
+			logrus.Debugf("Credentials not found")
```

Comment:
Could we remove this log. It's redundant to the one in line 88 and there's a potential for confusion.

Reply:
Well he returns right away, so line 88 will never happen?

Reply:
Thanks for clarifying, @rhatdan. Must have been the Monday morning...